### ISSUE TITLE
"make update" command failing

### ISSUE THREAD

**Issue by bartonstanley** [BlueprintKansas/ksvotes.org#160]
When following the instructions in README.md the `make update` step failed, see traceback below. 

Have not yet found a workaround for this.

(venv) $ make update
python manage.py db migrate
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
Traceback (most recent call last):
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/base.py", line 143, in _catch_revision_errors
    yield
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/base.py", line 206, in get_revisions
    return self.revision_map.get_revisions(id_)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 299, in get_revisions
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 299, in <listcomp>
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 304, in get_revisions
    for rev_id in resolved_id)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 304, in <genexpr>
    for rev_id in resolved_id)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 362, in _revision_for_ident
    resolved_id)
alembic.script.revision.ResolutionError: No such revision or branch 'ab29754d64e2'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "manage.py", line 53, in <module>
    manager.run()
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/flask_script/__init__.py", line 417, in run
    result = self.handle(argv[0], argv[1:])
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/flask_script/__init__.py", line 386, in handle
    res = handle(*args, **config)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/flask_script/commands.py", line 216, in __call__
    return self.run(*args, **kwargs)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/flask_migrate/__init__.py", line 197, in migrate
    version_path=version_path, rev_id=rev_id)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/command.py", line 176, in revision
    script_directory.run_env()
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/base.py", line 427, in run_env
    util.load_python_file(self.dir, 'env.py')
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/util/pyfiles.py", line 81, in load_python_file
    module = load_module_py(module_id, path)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/util/compat.py", line 83, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "migrations/env.py", line 87, in <module>
    run_migrations_online()
  File "migrations/env.py", line 80, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/runtime/environment.py", line 836, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/runtime/migration.py", line 321, in run_migrations
    for step in self._migrations_fn(heads, self):
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/command.py", line 156, in retrieve_migrations
    revision_context.run_autogenerate(rev, context)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/autogenerate/api.py", line 415, in run_autogenerate
    self._run_environment(rev, migration_context, True)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/autogenerate/api.py", line 425, in _run_environment
    if set(self.script_directory.get_revisions(rev)) != \
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/base.py", line 206, in get_revisions
    return self.revision_map.get_revisions(id_)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/contextlib.py", line 130, in __exit__
    self.gen.throw(type, value, traceback)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/base.py", line 174, in _catch_revision_errors
    compat.raise_from_cause(util.CommandError(resolution))
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/util/compat.py", line 194, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=exc_value)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/util/compat.py", line 187, in reraise
    raise value.with_traceback(tb)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/base.py", line 143, in _catch_revision_errors
    yield
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/base.py", line 206, in get_revisions
    return self.revision_map.get_revisions(id_)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 299, in get_revisions
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 299, in <listcomp>
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 304, in get_revisions
    for rev_id in resolved_id)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 304, in <genexpr>
    for rev_id in resolved_id)
  File "/Users/bartonstanley/Sandbox/ksvotes.org/venv/lib/python3.7/site-packages/alembic/script/revision.py", line 362, in _revision_for_ident
    resolved_id)
alembic.util.exc.CommandError: Can't locate revision identified by 'ab29754d64e2'
make: *** [dbmigrate] Error 1

**Comment by karpet** [comment]
What is in your `migrations/versions/` dir?

I see:
```
$ ls -l migrations/versions/
total 40
-rw-r--r--  1 karpet  staff   655 Jun 29 20:11 527e8f16751e_.py
-rw-r--r--  1 karpet  staff   843 Jun 14 22:18 879fe8a73df4_.py
-rw-r--r--  1 karpet  staff   847 Jun  4 18:24 8c3766231a8f_.py
drwxr-xr-x  7 karpet  staff   238 Jul 15 17:00 __pycache__/
-rw-r--r--  1 karpet  staff  2040 May 23 15:16 a4181f33a8e3_.py
-rw-r--r--  1 karpet  staff   671 Jul 15 17:00 d84e3212bc7b_.py
```

**Comment by karpet** [comment]
When you connect directly to your db, what version does it think it is?
```
$ psql -d ksvotes_dev -h localhost
ksvotes_dev=# select * from alembic_version ;
 version_num
--------------
 d84e3212bc7b
(1 row)
```

**Comment by karpet** [comment]
I think this is addressed in #274 -- for new setups, it should be `make dbupgrade` not `make update`.
